middleware.go

```diff
@@ -16,6 +16,25 @@ import (
 
 var defaultMetricPath = "/metrics"
 
+// RequestCounterURLLabelMappingFn is a function which can be supplied to the middleware to control
```

Comment:
this probably warrants a block comment with /* */

Reply:
Agreed, and done.